server_side/Const.py

```diff
@@ -0,0 +1,20 @@
+
+REQUEST_TYPE = 'protocol_type'
+USER_ID = 'user_id'
+TIME = 'time'
+NOT_FOUND = 'Not Found'
+
+# consts for requests from app and arduino:
+GET_HEARTBEAT = 'getHeartbeat'
```

Comment:
Why some of the constants are snake case  format "a_b_c" and some camel case "aBaCa?